src/components/ApiSearchList.jsx

```diff
@@ -0,0 +1,41 @@
+// Component
+// Search anime based on user input
+// Makes fetch request on submit
+// Returns informations
+
+import React, { useContext, useState } from "react";
+import { ApiContext } from "../contexts/ApiProvider";
+
+export function ApiSearchList() {
+  const { url } = useContext(ApiContext);
+  const { animeList, setAnimeList } = useState([]); // Use local state
+  const [searchData, setSearchData] = useState("");
+
+  const searchForAnime = async () => {
+    try {
+      const response = await fetch(url + "anime/" + searchData); // Update the API endpoint for anime
```

Comment:
it will only take an id as input atm. take a look at the getAnimeSearch endpoint in the docs. you will need to add a query parameter to the end of the anime word and drop the  '/'. e.g. `anime?sfw&q=`. this way when users input a string it will search for that specifically.

https://docs.api.jikan.moe/#tag/anime/operation/getAnimeSearch